iron-list.html

```diff
@@ -817,6 +817,10 @@
           parseInt(styles['padding-top'], 10);
       this._isRTL = Boolean(styles.direction === 'rtl');
       this._viewportWidth = this.$.items.offsetWidth;
+      var viewportHeight = this._scrollTargetHeight;
+      if (viewportHeight !== this._viewportHeight) {
+        this._resetScrollPosition(this.scrollTarget.scrollTop);
+      }
```

Comment:
I prefer that this be removed in favor of keeping the width setter in the test instead.